app/jobs/parse_report_job.rb

```diff
@@ -19,7 +19,7 @@ def perform(file, message)
   end
 
   def cancelled?
-    Sidekiq.redis { |c| c.exists("cancelled-#{jid}") }
+    Sidekiq.redis { |c| c.exists?("cancelled-#{jid}") }
```

Comment:
This fixes some deprecation warnings. There are still more within sidekiq itself.